admin/tool/policy/lang/en/tool_policy.php

```diff
@@ -27,8 +27,12 @@
 
 $string['addpolicydocument'] = 'Add new policy document';
 $string['addpolicyversion'] = 'Add new version';
+$string['age'] = 'Age';
+$string['agelocationverification'] = 'Age and location verification';
 $string['ageofdigitalconsentmap'] = 'Age of digital consent';
 $string['ageofdigitalconsentmap_desc'] = 'Allows to define the age of digital consent. Users below the age are not allowed to accept the policy documents on their own. In such a case, policies must accepted by the holder of parental responsibility over the user. The age may be different in different countries. If there is no explicit value provided for the user\'s country code, the wildcard character\'s value is used.';
+$string['countryofresidence'] = 'Country of residence';
```

Comment:
As discussed during the meeting, these strings must use a simple language, they are aimed at youngest kids. Let us not put "Country of residence". Better something like "Where do you live?" or so.